include/graaflib/graph.h

```diff
@@ -218,6 +228,18 @@ using directed_graph = graph<VERTEX_T, EDGE_T, graph_type::DIRECTED>;
 template <typename VERTEX_T, typename EDGE_T>
 using undirected_graph = graph<VERTEX_T, EDGE_T, graph_type::UNDIRECTED>;
 
+/**
+ * Get transposed version of a given directed graph
+ *
+ * @param  graph The directed graph that is to be transposed
+ * @param  out_transposed_graph An out parameter that will contain the
+ * transposed graph
+ */
+template <typename VERTEX_T, typename EDGE_T>
+void get_transposed_graph(
+    const directed_graph<VERTEX_T, EDGE_T>& graph,
+    directed_graph<VERTEX_T, EDGE_T>& out_transposed_graph);
```

Comment:
Instead of having `out_transposed_graph` an output parameter, I would rather use a return value:

```c++
template <typename V, typename E>
directed_graph<V, E> get_transposed_graph(const directed_graph<V, E>& graph) {...}
```

Reply:
No problem! Will make this tweak